staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -152,6 +153,15 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 				options,
 			)
 		}
+		if accessor, accessorErr := meta.Accessor(obj); accessorErr == nil {
+			// Dedup owner references before updating managed fields
```

Comment:
I did not put this before creating admission attributes because I don't think any admission plugin cares about the duplication, so let's keep it as it is. 

I don't think we should put this before audit logging because audit log can help identify problematic requests.